### ISSUE TITLE
Renderer should accept colors as parameters

### ISSUE THREAD

**Issue by franzheidl** [bpmn-io/bpmn-js#713]
Currently, the renderer renders most elements explicitly as black by default.

**It would be a great improvement if the renderer would be more generic and let the context decide how the diagram should be rendered, or default sensibly.**

There’s no need to hardcode 'black' in the renderer, SVG elements that have a stroke defined will always render black by default.

There should be an option to pass desired colors as parameters to the renderer, like 

`var viewer = new BpmnViewer({ container: 'body' , color: '#333333', background: '#ffffff', hover: 'magenta'});`

In case no or not all parameters for colors have been passed to the renderer, we should have a defaults object to use instead of having black hardcoded in the renderer itself.

Also, since bpmn allows to defgine colors in the xml, ideally there should be an option whether to render/respect this information or not.

**Comment by deysudip** [comment]
I have found a solution to render XML defined color in SVG, you have to tweak a function in source code a little.
`getFillColor(element, defaultColor)` & `getStrokeColor(element, defaultColor)`.

You have to change the attribute name passed here in `bo.get()`. I have changed it to `bo.get('color:background-color')` for `getFillColor(element, defaultColor)` and thus solving the issue.

**Comment by nikku** [comment]
We don't plan to perform changes here any time soon.

**Comment by nikku** [comment]
Closed via dc1f18a4972cee2d9e1bf46a6412dbb93b28506a.

You may now define default colors for elements via the `renderer` option:

```
var bpmnJS = new BpmnJS({
  renderer: {
    defaultFillColor: 'green',
    defaultStrokeColor: 'yellow'
  }
});
```

**Comment by franzheidl** [comment]
👍🙏
